### PR TITLE
click group decorator

### PR DESCRIPTION
**Pull request by volodymyrss**

### CODE REVIEW

**Review by Ph0tonic — APPROVED** [review]

**Comment by volodymyrss** [comment]
Please rather not merge it next time. The assigned person is in charge of merging. Maybe I realized something is missing and need to push and re-request.

**Comment by volodymyrss** [comment]
Another alternative is to let maintainer merge. Since we both maintain here, it's better if the assigned one merges for the reasons above.

**Pull request merged**
